pkg/cmd/roachtest/tests/knex.go

```diff
@@ -111,6 +114,31 @@ func registerKnex(r registry.Registry) {
 		)
 		rawResultsStr := result.Stdout + result.Stderr
 		t.L().Printf("Test Results: %s", rawResultsStr)
+		if err != nil {
+			// We don't have a good way of parsing test results from javascript, so we
```

Comment:
😬

Reply:
yeah it's not ideal. i couldn't find any libraries out there that are meant for parsing these results either, last i checked.